pkg/framework/plugins/nodeutilization/highnodeutilization_test.go

```diff
@@ -547,7 +533,7 @@ func TestHighNodeUtilization(t *testing.T) {
 	}
 }
 
-func TestValidateHighNodeUtilizationStrategyConfig(t *testing.T) {
+func TestValidateHighNodeUtilizationPluginConfig(t *testing.T) {
```

Comment:
Should these be moved to `pkg/validation/validation_pluginargs_test.go`?

Reply:
good idea, i update as your mention, please check